host/frontend/host-orchestrator/instancemanager.go

```diff
@@ -14,12 +14,36 @@
 
 package main
 
-type InstanceManager struct{}
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"sync"
+)
+
+type InstanceManager struct {
+	fetchCVDHandler *FetchCVDHandler
+}
+
+func NewInstanceManager(fetchCVDHandler *FetchCVDHandler) *InstanceManager {
+	return &InstanceManager{fetchCVDHandler}
+}
 
 func (m *InstanceManager) CreateCVD(req *CreateCVDRequest) (*Operation, error) {
 	if err := validateRequest(req); err != nil {
 		return nil, err
 	}
+	// This logic isn't complete yet, it's work in progress.
+	go func() {
```

Comment:
Can this give the caller some kind of future/promise object in the returned `Operation` instance? It's not clear what a client can do the with the `Operation` instance as-is.

Reply:
true, this logic is still work in progress, the next pr is about operation management which will give us an id to keep track of the returned operation.